modules/microshift-restoring-data-backups.adoc

```diff
@@ -0,0 +1,41 @@
+//Module included in the following assemblies:
+//
+// * microshift_updating/microshift-update-options.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-restoring-data-backups-manually_{context}"]
+= Restoring application data backups manually
+
+You can restore application data from a backup manually. Backups can be restored after updates, or after other system events that remove or damage required data. Backups are in the `/var/lib/microshift-backups` directory by default. When restoring a backup, you must use the entire file path.
+
+[NOTE]
+====
+RPM-OSTree systems backup and restore data automatically.
+====
+
+.Prerequisites
+* Root access to the host.
+* Full path of the data backup file.
+* {product-title} is stopped.
+
+[IMPORTANT]
```

Comment:
Should this note be moved to the Backing Up MicroShift data manually section ?

Reply:
It's there already.